vars/buildDockerAndPublishImage.txt

```diff
@@ -1,15 +1,50 @@
 <p>
```

Comment:
not really sure what the point of a .txt file is for documentation that includes html?

shouldn't this be ascii doc or markdown?

Reply:
My understanding is that it is used in the UI, in the "Pipeline Syntax" section to document the available libraries.

Reply:
Yes, it will be run through the configured `Jenkins.markdownFormatter`, but currently this file extension is expected by `workflow-cps-global-lib`.

Reply:
huh never knew about it.

Unfortunately it's quite buried and doesn't work so well with organisation folders, i.e. you have to actually go to a branch that has run, not very discoverable,

It's not in pipeline syntax but in Global Variables reference:
https://ci.jenkins.io/job/Plugins/job/abap-ci-plugin/job/master/pipeline-syntax/globals#buildPlugin

documented here:
https://www.jenkins.io/doc/book/pipeline/shared-libraries/#directory-structure

Reply:
and FYI infra.ci.jenkins.io doesn't seem to have a markup formatter configured, all the output just comes as plain text

Reply:
> you have to actually go to a branch that has run

Yes because Jenkins has to look this up from an actual build record.